assembly.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly
+  xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd"
+>
+  <id>zip-theme-jar</id>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <fileSets>
+    <fileSet>
+      <outputDirectory>.</outputDirectory>
+      <directory>${project.basedir}/build_theme</directory>
+      <includes>
+        <include>keycloak-theme-for-kc-22-to-25.jar</include>
+      </includes>
```

Comment:
Can we add the ability to configure targeted Keycloak versions using the config option `keycloakVersionTargets` based on a property `keycloakVersion(s)`, see more [here](https://docs.keycloakify.dev/targeting-specific-keycloak-versions)

And maybe rename the final name to `keycloak-theme-faimer-<keycloakVersion>-<projectVersion>`

Reply:
Yeah we can but how to make sure that the `projectVersion` is updated in `vite.config.ts` every time there is a change in `pom.xml` version

Reply:
Maven filtering via the Maven resource plugin to substitute at build time.